pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -198,9 +198,14 @@ func (rc *reconciler) unmountVolumes() {
 func (rc *reconciler) mountAttachVolumes() {
 	// Ensure volumes that should be attached/mounted are attached/mounted.
 	for _, volumeToMount := range rc.desiredStateOfWorld.GetVolumesToMount() {
-		volMounted, devicePath, err := rc.actualStateOfWorld.PodExistsInVolume(volumeToMount.PodName, volumeToMount.VolumeName)
+		volMounted, devicePath, err := rc.actualStateOfWorld.PodExistsInVolume(volumeToMount.PodName, volumeToMount.VolumeName, volumeToMount.SELinuxLabel)
 		volumeToMount.DevicePath = devicePath
-		if cache.IsVolumeNotAttachedError(err) {
+		if cache.IsSELinuxMountMismatchError(err) {
```

Comment:
Shouldn't we modify `reconstruct_new.go` file from reconstruction PR, rather than here? I want to see how it fits together.

Reply:
I removed all reconstruction changes from this PR, indeed it will need far more changes than presented here.